dist/joii.js

```diff
@@ -778,6 +787,19 @@
                 case 'immutable':
                     metadata.is_read_only = true;
                     break;
+                case 'SerializableAttribute':
```

Comment:
Fair enough. I gave a couple ways of doing it, since "SerializableAttribute" is the standard in C# (so people coming from that background would be familiar), whereas the others are just easier to type. I can narrow it down to just simply "serializable" though (or maybe "serialize" since it's shorter?) . What about the opposite (nonserializable or noserialize?).

Keep in mind - public properties are serializable by default (without needing to specify), whereas private/protected aren't.